fluent-kafka-streams-tests/src/test/java/com/bakdata/fluent_kafka_streams_tests/CountInhabitantsWithAvroTest.java

```diff
@@ -24,22 +24,31 @@
 
 package com.bakdata.fluent_kafka_streams_tests;
 
-import com.bakdata.fluent_kafka_streams_tests.test_applications.CountInhabitantsWithAvro;
+import com.bakdata.fluent_kafka_streams_tests.junit5.test_applications.CountInhabitantsWithAvro;
```

Comment:
This looks like a refactoring error. There should be no junit5 in this package name.

Reply:
Idd